File: 백준/Silver/11399. ATM/ATM.java
import java.io.*;
import java.util.Arrays;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());
        int[] times = new int[n];
        int ans = 0;
        int waiting_time = 0;
        String[] inputs = br.readLine().split(" ");

        for (int i=0; i<n; i++) {
            times[i] = Integer.parseInt(inputs[i]);
        }

        Arrays.sort(times);

        for (int time : times) {
            waiting_time += time;
            ans += waiting_time;
        }

        bw.write(ans+"");
        bw.flush();
    }

}

File: 백준/Silver/11399. ATM/ATM.py
import sys

if __name__ == "__main__":

    n = int(sys.stdin.readline())
    lst = list(map(int, sys.stdin.readline().split()))

    lst.sort()
    ans = 0
    waiting_time = 0
    for time in lst:
        waiting_time += time
        ans += waiting_time

    print(ans)